docs/design/1_hedera_etl.md

```diff
@@ -17,6 +17,7 @@ BigQuery's storage engine (ColumnIO) and processing engine (Dremel) were designe
 So the schema would be natural adaptation of Hedera's protobuf (with few deviations, mentioned below) and would look like:
 ```
 transactions (table name, not part of column names)
+├── consensusTimestampTruncated: int
 ├── consensusTimestamp: int
 ├── transactionType: int
 ├── entity:
```

Comment:
This field is named entityID in schema.json.

Reply:
ahh, thanks for catching. Fixed in schema.json